src/tests/functions_test.ts

```diff
@@ -1,6 +1,74 @@
 import { validateAndCreateFunctions } from "../functions.ts";
-import { assertExists } from "../dev_deps.ts";
+import { assertEquals, assertExists, assertRejects } from "../dev_deps.ts";
+import { Options } from "../types.ts";
 
 Deno.test("validateAndCreateFunctions", () => {
   assertExists(validateAndCreateFunctions);
 });
+
+Deno.test("Function files should have a default export", async () => {
```

Comment:
I am a bit of a pedant when it comes to test names - I really like for the test name to make clear what the assumptions around a test are, as well as what the expected behaviour is. I'd change this to something like:

```suggestion
Deno.test("validateAndCreateFunctions should not throw an exception when fed a function file that has a default export", async () => {
```